terraform/gcp/k8s/environments/prod.yaml

```diff
@@ -5,6 +5,7 @@ deploymentZone: prod
 replicas: 3
 dsmUrl: https://dsm.datadonationplatform.org/dsm
 dsmIssuer: juniper.terra.bio
+gcsFileStorageBucketName: juniper-participant-documents-prod
```

Comment:
how does auth work?  is the application automatically allowed to write in the bucket because it's running in the same project?  If so, I'd add some comments specifying that

Reply:
I'm not entirely sure how it works locally, but I can speak to how (I believe) it would work on the GKE side. Our pod is authenticated to a service account (i think it's called `juniper-app`) which is how it can ready secrets and stuff. I don't think it has enough perms, though, becuase I think it needs `roles/storage.objectUser` (see [docs](https://cloud.google.com/storage/docs/access-control/iam-roles), change needed in juniper_service_account.tf). Here you can see them testing out authetnication from a [pod](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#verify) with the same kind of authentication we use - it looks like the auth is injected into the pod itself, so (I think) it should automatically be picked up.